### PR TITLE
Bring back learning page.

### LINKED ISSUE #5825: Bring back Books + Videos page
Personally I'm not a big fan of the latest learning page: https://docs.pymc.io/en/latest/learning.html

It has too much text, lacks images, and the structure is quite confusing.

So I think we should bring back the old Learning page (Books + Videos): https://docs.pymc.io/en/v3/learn.html

### PR DESCRIPTION
**Pull request by twiecki**
image

Closes https://github.com/pymc-devs/pymc/issues/5825.

### CODE REVIEW

**Comment by twiecki** [comment]
Followed a different approach now reusing the existing structure.

**Comment by OriolAbril** [comment]
related to #5675 too, but should not close the issue yet until we update the sponsor benefits in the governance to match what we are doing in practice

**Review by OriolAbril — APPROVED** [review]

File: docs/source/learn/books.md [hunk 1]
```diff
@@ -1,74 +1,95 @@
...
+::::{grid} 1 2 2 3
+:gutter: 3
 
-## Bayesian Methods for Hackers
+:::{grid-item-card} Bayesian Methods for Hackers
 
 By Cameron Davidson-Pilon
 
+<img src="https://camo.githubusercontent.com/4a0aca82ca82efab71747d00db30f3a68de98e82/687474703a2f2f692e696d6775722e636f6d2f36444b596250622e706e673f31">
+
```
Comment (OriolAbril) [inline docs/source/learn/books.md]:
```suggestion
:::{grid-item-card} Bayesian Methods for Hackers
:img-top: https://camo.githubusercontent.com/4a0aca82ca82efab71747d00db30f3a68de98e82/687474703a2f2f692e696d6775722e636f6d2f36444b596250622e706e673f31

By Cameron Davidson-Pilon

```

Comment (twiecki) [reply docs/source/learn/books.md]: Should I change all of them?

Comment (OriolAbril) [reply docs/source/learn/books.md]: If you want them to look like the ArviZ ones yes!

**Comment by OriolAbril** [comment]
You have to update https://github.com/pymc-devs/pymc/blob/main/docs/source/conf.py#L157 too

**Comment by twiecki** [comment]
Thanks for fixing it up!

**Pull request merged**